src/libs/actions/Report.js

```diff
@@ -1138,7 +1138,16 @@ function addAction(reportID, text, file) {
     const parser = new ExpensiMark();
     const commentText = parser.replace(text);
     const isAttachment = _.isEmpty(text) && file !== undefined;
-    const attachmentInfo = isAttachment ? file : {};
+    let attachmentInfo = {};
```

Comment:
Just add source to the file object 
https://github.com/Expensify/App/blob/853db572e36748541ec6c809d12a3ff2b61be8b5/src/components/AttachmentModal.js#L115

``` _.extend(file, {source: this.state.sourceURL});```